src/novaUtils.ts

```diff
@@ -56,7 +56,11 @@ export function jumpToRange(
   });
 }
 
-export const makeServerExecutable = () => {
+export const makeServerExecutable = (path: string) => {
+  if (!path || path === "") {
```

Comment:
Is this check correct? This seems to be reversed. If the path is **not** set we want to use the default server path and update the permissions. If the path is set we want to return early.